src/dsp_tools/excel2xml.py

```diff
@@ -2041,17 +2043,20 @@ def _convert_resource_row_to_xml(
     resource_id = row["id"]
     resource_label = row.get("label")
     if pd.isna([resource_label]):
-        raise BaseError(f"Missing label for resource '{resource_id}' (Excel row {row_number})")
-    if not check_notna(resource_label):
+        resource_label = ""
+        warnings.warn(f"Missing label for resource '{resource_id}' (Excel row {row_number})")
```

Comment:
Do you really want to make the label optional? That seems not like a good idea since the label is used in the APP